pyccl/nl_pt/power.py

```diff
@@ -580,10 +581,16 @@ def get_pt_pk2d(cosmo, tracer1, tracer2=None, ptc=None,
                         lk_arr=np.log(ptc.ks),
                         pk_arr=p_pt[1].T,
                         is_logp=False)
-        return pt_pk_ee, pt_pk_bb
+        if return_ptc:
```

Comment:
Could we add a quick unit test for this

Reply:
Should these go in `pyccl/tests/`? or in `benchmarks/`?
@damonge @c-d-leonard

Reply:
`pyccl/tests/`